builtin/logical/aws/secret_access_keys.go

```diff
@@ -125,6 +125,9 @@ func (b *backend) assumeRole(ctx context.Context, s logical.Storage,
 		RoleArn:         aws.String(roleArn),
 		DurationSeconds: &lifeTimeInSeconds,
 	}
+        if roleSessionName != "" {
+                assumeRoleInput.SetRoleSessionName(roleSessionName)
+        }
```

Comment:
Indentation is a little off on these. Would you be willing to run `$ make fmt` from the Vault home directory, or `$ go fmt` on each of these files?

Reply:
ran `$ go fmt`